### DISCUSSION TITLE
OpenSSL error when using copilot on linux machine remote

### DISCUSSION THREAD

**Discussion by ysig** [community/community#43736] [Copilot Conversations]
Hi,

I get the following error when using the **copilot** extension,

`[ERROR] [default] [2023-01-05T14:34:00.913Z] GitHub Copilot could not connect to server. Extension activation failed: "tunneling socket could not be established, cause=write EPROTO 140473655486400:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:`

Is there a way to resolve it?

**Comment by iamcjwilliams** [comment]
@ysig Sorry for the trouble here, Could you provide a few more details on how this can be replicated?

**Reply by ysig** [reply]
How can I provide more details? It's on the server that I'm working and it pops out this error message. The question is what causing it and how I can fix it and investigate it.

**Reply by ysig** [reply]
My current openssl version is: OpenSSL 1.1.1k  25 Mar 2021

**Comment by iamcjwilliams** [comment]
@ysig Try [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue
